src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2083,19 +2083,18 @@ limitations under the License.
         </clusters>
     </deviceType>
     <deviceType>
-        <name>MA-boolean-sensor</name>
+        <name>MA-water-freeze-detector</name>
```

Comment:
It looks like we are missing the rain sensor here?

Reply:
Missed it. Will include it, thanks.